emulators.tex

```diff
@@ -8,14 +8,127 @@ \chapter{Emulators}
 
 Download the MEGA65 emulator source code from \url{https://github.com/lgblgblgb/xemu}.
 
-Download pre-compiled versions from \url{https://github.lgb.hu/xemu/}.
+Download pre-compiled versions from \url{https://github.lgb.hu/xemu/}. Installers are
+available for Apple Macintosh, Windows and Linux.
 
-A live ISO image containing the emulator, documentation, and other tools is available from \url{Forum64.de}
+A live ISO image containing the emulator, documentation, and other tools is available from
+Forum64.de
 at \url{https://www.forum64.de/index.php?thread/104698-xemu-live-system-iso-file/\&postID=1549927\#post1549936}.
+Should you choose to use it you may find instructions on page \pageref{sec:live-iso-image}.
 
 \section{Using The Xmega65 Emulator}
+\index{emulator}
+
+\subsection{The keyboard}
+\index{keyboard}
+
+When using a real Mega 65 a keyboard layout related to the old Commodore machines is
+encountered. How can this be adapted to a PC/Mac ? It's not that hard. Most of the keys
+for writing stay the same - given an American layout. So also special characters reached
+by \specialkey{Shift} and a number are same to a U.S. keyboard.
+
+The closer you get to \specialkey{Enter} or \widekey{Backspace} keys you will find
```

Comment:
The six keys this refers to are not above the cursor keys in all cases, even though they are in the diagram. Let's be direct:

"Some keys are only present on the MEGA65 keyboard and have not PC equivalents. Xmega65 uses the PC keys \specialkey{Insert}, \specialkey{Delete}, \specialkey{Home}, \specialkey{End}, \specialkey{PageUp}, and \specialkey{PageDown} to represent \specialkey{+}, \specialkey{\pound}, \specialkey{Clr/Home}, \specialkey{Run/Stop}, \specialkey{Help}, and \widekey{Restore}, respectively."

(Double-check my LaTeX here, I haven't confirmed which key macros to use.)

Reply:
> &hellip;are not above the cursor keys in all cases&hellip;

@dansanderson If not Xemu's positional scancode routine would not work I fear. Your double key item list would probably be understood if put in a table. I thought both to be obvious in the layout if verbally explained. No worries, again we use your text.

The pound symbol key does not work in my environment. It is the only key not using a keyword but `\pounds` as a Tex command. I did it as found in other files of the project. I'm not sure whether those files would render it correct in my setup. Will have to test. But not today.

Reply:
I have doc builds working on my end (or at least I think I do :) ) and can do a test build before merging, if that helps.

Reply:
I found `\pounds` perfectly working in the fat Mega Book PDF I built in my env. Strange it doesn't as the Dev Book PDF. Maybe both use different Tex environment settings.

![grafik](https://user-images.githubusercontent.com/67829169/204220998-de9c3cac-8fcb-4768-88ef-1671fbf25710.png)

Reply:
I can repro that it's not working in the Emulators chapter but is working in the keyboard reference, both in the Book. (And also it's not working in the Dev Guide.) I don't know enough about how the fonts work. Maybe we can ask the Discord for help.

![CleanShot 2022-11-28 at 10 29 13](https://user-images.githubusercontent.com/408973/204353335-7d2620f9-6bcf-4bfe-8bd9-60121d010b11.png)

![CleanShot 2022-11-28 at 10 37 37](https://user-images.githubusercontent.com/408973/204354705-9d664105-9466-4d01-8f03-26cdc6426187.png)

Reply:
@dansanderson Okay, if you and I now both are able to reproduce I don't need to see what Travis had produced.

Reply:
@MirageBD fixed the missing glyph in the font in https://github.com/MEGA65/mega65-user-guide/pull/515, so this is now working!